src/main/kotlin/racing/race/CarRace.kt

```diff
@@ -0,0 +1,19 @@
+package racing.race
+
+import racing.common.CarRaceProperty
+import racing.view.ResultView
+
+class CarRace(
+    private val carRaceProperty: CarRaceProperty,
+    private val resultView: ResultView,
+) {
+    fun start() {
+        for (round in CarRaceProperty.ROUND_START..carRaceProperty.roundSize) {
+            carRaceProperty.cars.map { it.move() }
```

Comment:
리턴값이 불필요할땐 forEach를 사용해도 됩니다!
```suggestion
            carRaceProperty.cars.forEach { it.move() }
```